tasks/services/db.js

```diff
@@ -2,9 +2,15 @@ const { Pool } = require("pg");
 
 class DB {
     async getTasksDB() {
+        // Support for local postgres db connection
+        let connectString = process.env.TASKS_DATABASE_URL;
+        if(connectString === undefined && process.env.NODE_ENV !== "production"){
```

Comment:
- Lets maintain TASKS_DATABASE_URL as is. 
- Delete the code here to what it was
- You can set TASKS_DATABASE_URL in docker-compose.dev.yml with the string
postgres://postgres:postgres@localhost:5432/btw
- Did you test if this is working. DB_host I think is "db" and not "localhost"